examples/all-clusters-app/esp32/sdkconfig.defaults

```diff
@@ -30,6 +30,7 @@ CONFIG_ESPTOOLPY_MONITOR_BAUD=115200
 #enable BT
 CONFIG_BT_ENABLED=y
 CONFIG_BT_NIMBLE_ENABLED=y
+CONFIG_BT_NIMBLE_HOST_BASED_PRIVACY=y
```

Comment:
We do not use host based privacy for ESP32C3. Do you want to create separate sdkconfig.defaults for ESP32 and ESP32C3? Also, with that, the call for `ble_hs_pvcy_rpa_config` in `src/platform/ESP32/nimble/BLEManagerImpl.cpp` should be added under `CONFIG_BT_NIMBLE_HOST_BASED_PRIVACY`

Reply:
fixed